README.md

```diff
@@ -356,6 +356,28 @@ Source code and an Eclipse project for the Flexy Java app are made available in
 #### Development Environment
 The Flexy Java application component was developed using a standard Ewon Java development environment. Documentation and additional information about the Ewon Java development environment is available in the Ewon Java Toolkit User Guide \(J2SE\) at [https://developer.ewon.biz/content/java-0#dev-documents](https://developer.ewon.biz/content/java-0#dev-documents).
 
+### Cloning
+
+Using the git client of your choice, clone the https://github.com/hms-networks/flexy-thingworx-connector repository.
+
+Using HTTPS:
+```console
+> git clone https://github.com/hms-networks/flexy-thingworx-connector.git --recursive
+```
+
+Using SSH:
+```console
+> git clone git@github.com:hms-networks/flexy-thingworx-connector.git --recursive
+```
```

Comment:
If a user doesn't know how to clone should there be a description here of what cloning is?

Reply:
Is the intent that all users will be cloning the repo or will most users just grab release packages? If most users should just grab releases you might want to clarify that cloning the repo is only for development and unnecessary for most users.